src/pages/BlockDetail/BlockComp.tsx

```diff
@@ -296,11 +297,32 @@ export const BlockComp = ({
   total: number
 }) => {
   const totalPages = Math.ceil(total / pageSize)
-  const { hash } = useLocation()
+  const { push } = useHistory()
+  const { hash, search } = useLocation()
+  const { param: blockId } = useParams<{ param: string }>()
+
+  const filter = new URLSearchParams(search).get('filter')
```

Comment:
It can replaced by useSearchParams.

Reply:
It's been replaced in https://github.com/Magickbase/ckb-explorer-frontend/pull/8/commits/df8664ca93e90eeed6194ff4c3080d06844a78ef